src/sentry/event_manager.py

```diff
@@ -662,6 +664,22 @@ def _pull_out_data(jobs, projects):
         )
 
 
+def _is_commit_sha(version: str):
+    return re.match(r"[0-9a-f]{40}", version) is not None
```

Comment:
Should this matching logic be a functionality that is shared in a vcs related functions?

What other VCS' do we support besides Git?
Do we always use the long commit? or do we also consider short commits?

Reply:
For the initial version, we'll only use long commits